code/modules/client/preference_setup/loadout/lists/belt.dm

```diff
@@ -0,0 +1,35 @@
+/datum/gear/belt
+	display_name = "general belt"
+	path = /obj/item/storage/belt
+	slot = slot_belt
+	sort_category = "Utility belts"
+	cost = 4
+
+/datum/gear/belt/tool
+	display_name = "tool belt"
+	path = /obj/item/storage/belt/utility
+	cost = 2
+
+/datum/gear/belt/nt_belt
+	display_name = "Absolutism utility belt"
+	path = /obj/item/storage/belt/utility/neotheology
+	cost = 2
+
+/datum/gear/belt/nt_belt
+	display_name = "Absolutism tactical belt"
+	path = /obj/item/storage/belt/security/neotheology
+	cost = 4
+
+/datum/gear/belt/mkbelt
+	display_name = "handmade tool belt"
+	path = /obj/item/storage/belt/utility/handmade
+	cost = 2
+
+/datum/gear/belt/mkbelt
```

Comment:
this is already present in the medical's loadout section and should be doc locked